strongbox-web-core/src/main/java/org/carlspring/strongbox/app/StrongboxSpringBootApplication.java

```diff
@@ -58,19 +52,4 @@ public static void restart()
         thread.setDaemon(false);
         thread.start();
     }
-
-    @Configuration
```

Comment:
Not needed now that the ACI is guaranteed to have set the `strongbox.storage.booter.basedir` up front.